data/udev/99-thrustmaster-wheel-perms.rules

```diff
@@ -10,5 +10,8 @@ SUBSYSTEMS=="hid", KERNELS=="0003:044F:B65E.????", DRIVERS=="t500rs", RUN+="/bin
 # Thrustmaster T300RS Racing Wheel (USB)
 SUBSYSTEMS=="hid", KERNELS=="0003:044F:B66E.????", DRIVERS=="t300rs", RUN+="/bin/sh -c 'cd %S%p/../../../; chmod 666 range spring_level damper_level friction_level'"
 
+# Thrustmaster T500RS Racing Wheel (USB)
+SUBSYSTEMS=="hid", KERNELS=="0003:044F:B65E.????", DRIVERS=="t500rs", RUN+="/bin/sh -c 'cd %S%p/../../../; chmod 666 range spring_level damper_level friction_level'"
+
```

Comment:
This part is already there above at line 7-8.

Reply:
Good catch, i just rebased and didn't check duplicity. Will update now

Reply:
Thanks, looks good to me now.